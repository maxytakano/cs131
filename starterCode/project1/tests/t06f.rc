function : int _foo()
{
	int x;
}