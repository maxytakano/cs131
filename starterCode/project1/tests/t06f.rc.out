Error, "/tmp/cs131_testrunner_iETaux/file.rc", line 4: 
  Return required in function, none found.
Compile: failure.
